Formula/liburing.rb

```diff
@@ -0,0 +1,30 @@
+class Liburing < Formula
+  desc "This is the io_uring library, liburing"
+  homepage "https://github.com/axboe/liburing"
+  url "https://github.com/axboe/liburing/archive/refs/tags/liburing-2.2.tar.gz"
+  sha256 "e092624af6aa244ade2d52181cc07751ac5caba2f3d63e9240790db9ed130bbc"
+  license "MIT"
```

Comment:
```
All software contained within this repo is dual licensed LGPL and MIT, see
COPYING and LICENSE, except for a header coming from the kernel which is
dual licensed GPL with a Linux-syscall-note exception and MIT, see
COPYING.GPL and <https://spdx.org/licenses/Linux-syscall-note.html>.
```

Reply:
Something like this than.
```suggestion
  license any_of: ["MIT", "LGL-3.0-only"]
```